demo/mobile/config.xml

```diff
@@ -1,93 +1,66 @@
+        <splash density="land-mdpi" src="www/res/splash/android/splash-land-mdpi.png" />
+        <splash density="land-xhdpi" src="www/res/splash/android/splash-land-xhdpi.png" />
+        <splash density="port-hdpi" src="www/res/splash/android/splash-port-hdpi.png" />
+        <splash density="port-ldpi" src="www/res/splash/android/splash-port-ldpi.png" />
+        <splash density="port-mdpi" src="www/res/splash/android/splash-port-mdpi.png" />
+        <splash density="port-xhdpi" src="www/res/splash/android/splash-port-xhdpi.png" />
+        <preference name="SplashScreen" value="screen" />
+    </platform>
+    <platform name="ios">
+        <icon height="180" src="www/res/icons/ios/Icon-60@3x.png" width="180" />
+        <icon height="60" src="www/res/icons/ios/Icon-60.png" width="60" />
+        <icon height="120" src="www/res/icons/ios/Icon-60@2x.png" width="120" />
+        <icon height="76" src="www/res/icons/ios/Icon-76.png" width="76" />
+        <icon height="152" src="www/res/icons/ios/Icon-76@2x.png" width="152" />
+        <icon height="40" src="www/res/icons/ios/Icon-40.png" width="40" />
+        <icon height="80" src="www/res/icons/ios/Icon-40@2x.png" width="80" />
+        <icon height="57" src="www/res/icons/ios/icon.png" width="57" />
+        <icon height="114" src="www/res/icons/ios/icon@2x.png" width="114" />
+        <icon height="72" src="www/res/icons/ios/Icon-72.png" width="72" />
+        <icon height="144" src="www/res/icons/ios/Icon-72@2x.png" width="144" />
+        <icon height="29" src="www/res/icons/ios/Icon-small.png" width="29" />
+        <icon height="58" src="www/res/icons/ios/Icon-small@2x.png" width="58" />
+        <icon height="50" src="www/res/icons/ios/Icon-50.png" width="50" />
+        <icon height="100" src="www/res/icons/ios/Icon-50@2x.png" width="100" />
+    </platform>
+    <plugin name="cordova-plugin-camera" spec="~2.3.0" />
     <plugin name="cordova-plugin-console" spec="~1.0.3" />
-    <plugin name="cordova-plugin-whitelist" spec="~1.2.2" />
-    <plugin name="cordova-plugin-splashscreen" spec="~4.0.0" />
-    <plugin name="cordova-plugin-network-information" spec="~1.2.1" />
+    <plugin name="cordova-plugin-device" spec="~1.1.2" />
     <plugin name="cordova-plugin-file" spec="~4.2.0" />
-    <plugin name="cordova-plugin-geolocation" spec="~2.4.0" />
     <plugin name="cordova-plugin-file-transfer" spec="~1.5.1" />
+    <plugin name="cordova-plugin-geolocation" spec="~2.4.0" />
     <plugin name="cordova-plugin-inappbrowser" spec="1.6.1" />
-    <plugin name="cordova-plugin-camera" spec="~2.3.0" />
+    <plugin name="cordova-plugin-network-information" spec="~1.2.1" />
+    <plugin name="cordova-plugin-splashscreen" spec="~4.0.0" />
+    <plugin name="cordova-plugin-whitelist" spec="~1.2.2" />
     <plugin name="phonegap-plugin-barcodescanner" spec="~6.0.0" />
-    <engine name="android" spec="~5.2.0" />
```

Comment:
We should keep the engine tags but without the spec so the platforms get added whe  `cordova prepare` is run
<engine name="android"/>
<engine name="ios"/>